joern-cli/frontends/kotlin2cpg/src/main/scala/io/joern/kotlin2cpg/ast/AstForDeclarationsCreator.scala

```diff
@@ -75,7 +75,7 @@ trait AstForDeclarationsCreator(implicit withSchemaValidation: ValidationMode) {
       classDeclarations.collectAll[KtProperty].filter(_.getInitializer != null).map { decl =>
         val initializerAsts = astsForExpression(decl.getInitializer, None)
         val rhsAst =
-          if (initializerAsts.size == 1) initializerAsts.head
+          if (initializerAsts.size == 1) initializerAsts.headOption.getOrElse(Ast())
```

Comment:
This is unnecessary and changes the outcome since this is already safe code. There is already a `initializerAsts.size == 1` guarantee. For style reasons perhaps it may make sense to do
```scala
initializerAsts.headOption.getOrElse(Ast(unknownNode(decl, "<empty>")))
```